web-ui/view/documentation/build_page.ml

```diff
@@ -54,11 +54,11 @@ let show =
     p
       [
         txt
-          "The running time of a build is defined to be the sum of the time \
-           taken by the analysis step and that of the longest running step of \
-           the build. It represents the 'wall clock' or elapsed time of the \
-           build and is different from the 'total build run time' in that it \
-           represents the time taken if all steps ran in parallel.";
+          "A build's running time is defined to be the sum of the time taken \
+           by the analysis step and that of the longest running step of the \
+           build. It represents the 'wall clock' or elapsed time of the build \
+           and is different from the 'total build run time,' as it represents \
```

Comment:
```suggestion
           and is different from the 'total build run time', as it represents \
```

Reply:
Ha! That's English typography, the apostrophe and comma are often swapped. It's stressful for us programmers. With proper kerning and font rendering, they should be nicely rendered one on top the other.
We could also use typographic quotes 🔥

Reply:
Oh thanks ^^

Reply:
The MLA style manual puts punctuation inside quotation marks, which look better when it's " instead of ', so if you want it outside of 'total build run time', instead of inside, I'll concede. 🙂

Reply:
No problem, let's keep it this way! Thanks for the information I'll go to bed smarter 😉